tests/test_monitor.py

```diff
@@ -167,3 +167,58 @@ def test_last_turns_monitor(test_context):
     assert np.all(monitor.x == np.array([[0,0,2,1,0],[3,5,7,9,11],[0,-2,-4,-6,-8],[20,23,26,29,32]]))
 
 
+@for_all_test_contexts
+def test_collective_monitor(test_context):
```

Comment:
To be renamed `test_collective_ebe_monitor`

Reply:
I've renamed the test as per the suggestion and added a check for None in the monitor definition.